ignite/contrib/handlers/clearml_logger.py

```diff
@@ -147,12 +148,14 @@ def __setattr__(self, attr: str, val: Any) -> None:
 
             self._task = _Stub()
         else:
-            self._task = Task.init(
-                project_name=kwargs.get("project_name"),
-                task_name=kwargs.get("task_name"),
-                task_type=kwargs.get("task_type", Task.TaskTypes.training),
-                **experiment_kwargs,
-            )
+            self._task = Task.current_task()
```

Comment:
Could you please leave a comment here on what we are trying to do like the PR's description:
"ClearMLLogger tries to retrieve current task before trying to create a new one." ?